packages/sit-onyx/src/components/OnyxFlyout/OnyxFlyout.stories.ts

```diff
@@ -39,8 +39,14 @@ type Story = StoryObj<typeof OnyxFlyout>;
 export const Default = {
   args: {
     label: "Choose application language",
-    default: ({ trigger }) => [
-      h(OnyxButton, { label: "Trigger", mode: "plain", color: "neutral", ...trigger }),
+    default: ({ trigger, disabled }) => [
+      h(OnyxButton, {
+        label: "Trigger",
+        mode: "plain",
+        disabled,
```

Comment:
Shouldn't we include the disabled in the trigger itself? So the user does not have to take about this on his own